This patch was developed in-house.  We have not yet decided whether to offer
it upstream or not.

--- Python-3.5.0a4/configure.ac.~3~	2015-04-20 14:10:42.056744800 -0700
+++ Python-3.5.0a4/configure.ac	2015-04-20 14:10:42.275328447 -0700
@@ -4412,7 +4412,7 @@
 
 AC_SUBST(EXT_SUFFIX)
 case $ac_sys_system in
-    Linux*|GNU*|Darwin)
+    Linux*|GNU*|Darwin|SunOS)
 	EXT_SUFFIX=.${SOABI}${SHLIB_SUFFIX};;
     *)
 	EXT_SUFFIX=${SHLIB_SUFFIX};;
